module/Application/src/Application/Service/AclFactory.php

```diff
@@ -47,6 +49,7 @@ public function createService(ServiceLocatorInterface $serviceLocator)
 		$acl->allow(User::ROLE_USER, 'Ora\Task', 'TaskManagement.Task.accept', new TaskOwnerAndCompletedTaskWithEstimationProcessCompletedAssertion());
 		$acl->allow(User::ROLE_USER, 'Ora\Task', 'TaskManagement.Task.assignShares', new TaskMemberAndAcceptedTaskAssertion());
 
+		$acl->allow(null, null, 'Application.Host.allowLocalhost', new HttpHostLocalhostAssertion());
```

Comment:
Non mi è chiaro... non voglio fare riferimento ad una risorsa in particolare, l'operazione a cui mi riferisco è quella di controllare se l'host è localhost... si potrebbe allora cambiare in `Application.Host.checkIfLocalhost` ?